src/frontend/src/components/WebformTable/components/EditTable/index.tsx

```diff
@@ -0,0 +1,64 @@
+import { Table as MuiTable, TableBody, TableHead } from "@material-ui/core";
+import React from "react";
+import { SAMPLE_EDIT_WEBFORM_METADATA_KEYS_TO_HEADERS } from "../../common/constants";
+import {
+  IsPrivateTableCell,
+  StyledTableCell,
+  StyledTableRow,
+} from "../../style";
+import Row from "../Row";
+import { TableProps } from "../UploadTable";
+
+export default function EditTable({
+  metadata,
+  handleRowMetadata,
+  applyToAllColumn,
+  handleRowValidation,
+  locations,
+}: TableProps): JSX.Element {
+  return (
+    <MuiTable component="div" stickyHeader>
+      <TableHead component="div">
+        <StyledTableRow {...({ component: "div" } as unknown)}>
+          <StyledTableCell component="div">
+            {SAMPLE_EDIT_WEBFORM_METADATA_KEYS_TO_HEADERS.privateId}
+          </StyledTableCell>
+          <StyledTableCell component="div">
+            {SAMPLE_EDIT_WEBFORM_METADATA_KEYS_TO_HEADERS.publicId}
+          </StyledTableCell>
+          <StyledTableCell component="div">
+            {SAMPLE_EDIT_WEBFORM_METADATA_KEYS_TO_HEADERS.collectionDate}
+          </StyledTableCell>
+          <StyledTableCell component="div">
+            {SAMPLE_EDIT_WEBFORM_METADATA_KEYS_TO_HEADERS.collectionLocation}
+          </StyledTableCell>
+          <StyledTableCell component="div">
+            {SAMPLE_EDIT_WEBFORM_METADATA_KEYS_TO_HEADERS.sequencingDate}
+          </StyledTableCell>
+          <IsPrivateTableCell align="center" component="div">
+            {SAMPLE_EDIT_WEBFORM_METADATA_KEYS_TO_HEADERS.keepPrivate}
+          </IsPrivateTableCell>
+        </StyledTableRow>
+      </TableHead>
+      {metadata && (
+        <TableBody component="div">
+          {Object.entries(metadata).map(([sampleId, sampleMetadata], index) => {
+            return (
+              <Row
+                isFirstRow={index === 0}
+                key={sampleId}
+                id={sampleId}
+                metadata={sampleMetadata}
+                handleMetadata={handleRowMetadata}
+                applyToAllColumn={applyToAllColumn}
+                handleRowValidation={handleRowValidation}
+                locations={locations}
+                skipIdColumn={true}
```

Comment:
```suggestion
                skipIdColumn
```

props default to `true`